packages/pycharm-community-bin/pycharm-community-bin.pacscript

```diff
@@ -0,0 +1,43 @@
+name="pycharm-community-bin"
+version="2021.3.3"
+pkgname="pycharm-community"
+gives="pycharm-community"
+maintainer="Emil Aarestrup <emil@aarestrup.nu>"
+description="Python IDE for professional developers aimed at productivity with a keyboard-centric focus."
+url="https://download.jetbrains.com/python/pycharm-community-${version}.tar.gz"
+hash="06ef4152d59b97da1186cdf288588d20d69a689fe75b2140130fa5d8326d07ad"
+
+desktop_files_path="$STOWDIR/$name/usr/share/applications"
+application_files_path="$STOWDIR/$name/usr/bin/$pkgname-$version"
```

Comment:
This would move files to `/usr/bin/pycharm-community-bin-2021.3.3/`. I don't think that's what you intended.

Reply:
I omitted the version number in the recent commit. As far as I understand, this is where other packages dump their files, so I assume this change is correct.